ifrs17/Utils/IdentityPropertyReader.ipynb

```diff
@@ -0,0 +1,80 @@
+{
+    "metadata": {
+        "authors": [],
+        "kernelspec": {
+            "display_name": "Formula Framework",
+            "language": "C#",
+            "name": "C#"
+        },
+        "language_info": {
+            "file_extension": ".cs",
+            "mimetype": "text/plain",
+            "name": "C#"
+        }
+    },
+    "nbformat": 4,
+    "nbformat_minor": 5,
+    "cells": [
+        {
+            "cell_type": "code",
+            "source": [
+                "#!import \"../DataModel/DataStructure\"",
+                "\nusing System.Linq.Expressions;",
+                "\nusing static Systemorph.Vertex.Equality.IdentityPropertyExtensions;"
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "code",
+            "source": [
+                "public class IdentityPropertyReader<T> where T:BaseVariableIdentity {",
```

Comment:
We need to improve this method: 
- allowing the user to be able to set the excluded properties and get the corresponding eventually updated instance;
- simplifying the concat logic.
We can work together to achieve both these goals.